test/test_pebble.py

```diff
@@ -27,8 +27,10 @@
 
 import pytest
 import websocket  # type: ignore
+from testing import Harness
 
-from ops import pebble
+import ops
+from ops import Framework, pebble
```

Comment:
This comment will be moot given this will be moving, but our preferred style for almost all imports is "import foo ... foo.bar()". Here we want access to `ops`, `pebble`, and `testing`, so we'd use:

```python
import ops  # for ops.CharmBase, ops.Framework
from ops import pebble, testing  # for pebble.Foo, testing.Harness
```